package.json

```diff
@@ -34,51 +33,47 @@
   },
   "homepage": "https://itowns.github.io/",
   "dependencies": {
-    "@mapbox/vector-tile": "^1.3.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/runtime": "^7.1.2",
+    "@mapbox/togeojson": "^0.16.0",
+    "@mapbox/vector-tile": "^1.3.1",
     "@tweenjs/tween.js": "^17.2.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-runtime": "^6.20.0",
+    "earcut": "^2.1.3",
     "flatbush": "^1.3.0",
     "js-priority-queue": "^0.1.5",
     "pbf": "^3.1.0",
-    "text-encoding-utf-8": "^1.0.1",
-    "togeojson": "^0.16.0",
-    "url-polyfill": "^1.0.8",
-    "whatwg-fetch": "^2.0.2"
-  },
-  "peerDependencies": {
-    "proj4": "^2.4.4",
+    "proj4": "^2.5.0",
+    "text-encoding-utf-8": "^1.0.2",
     "three": "^0.97.0",
```

Comment:
Why did it (along with three.meshline and proj4) move from a peer dependency to a "normal" dependency?

Fwiw, url-polyfill and whatwg-fetch should probably be dev dependencies (they're polyfills, and only used by the webpack build)

Reply:
>Why did it (along with three.meshline and proj4) move from a peer dependency to a "normal" dependency?

It seems that the peer dependencies are used in the development of plugin package. These dependencies (`Three, proj4`...) are necessary for webpack building when using `itowns` package in other project.
By example, `Itowns` isn't a `Three package plugin,` it's dependent on `Three` so it must be installed automatically.  Otherwise why not move all the itowns dependencies in the peerDependencies.

>Fwiw, url-polyfill and whatwg-fetch should probably be dev dependencies (they're polyfills, and only used by the webpack build)

yes indeed there will be no reason to leave in dependencies

Reply:
> It seems that the peer dependencies are used in the development of plugin package. These dependencies (`Three, proj4`...) are necessary for webpack building when using `itowns` package in other project.

See https://github.com/iTowns/itowns/commit/400d3d060940b4b63049678e2523c49df0a1b0eb (and the 2.2.0 release notes)
The problem is that as soon as you use a different version of Three (or proj4), NPM will no longer _hoist_ it, you'll use the two separate versions of Three at the same time, and of course any `instanceof` check will fail.

Reply:
Ok, but it's the same with the packages `tween, earcut, @mapbox/vector-tile, js-priority-queue, pbf`...?

Reply:
AFAICT, js-priority-queue (for example) is entirely internal to iTowns, so it does not really matter whether it appears twice. Three is exposed through the API of iTowns, so it's another matter (and iTowns uses `instanceof` on Three objects, so it's important that the user of iTowns and iTowns itself use the exact same version)

Reply:
ok, I see, thanks for the explanation

Reply:
I think we should not expose `THREE` anymore. Because he weighed down the package and it's not too much in the logic of the packages. WDYT?

Reply:
I'm even for removing three and proj4 from the exported package of itowns: it is up to the user to include three in its webpage. That may cause more problem on our side, being careful when upgrading a version of three, because a user could want a less/more recent version of three. I'm not pushing anything of course, as this opinion is subject to some lengthy discussions.

Reply:
It should be noted that `PeerDependencies` don't prohibit the use of different versions of THREE. They don't raise the problem of conflict with `instanceof`.